contracts/partial/parser/storage.ligo

```diff
@@ -8,6 +14,7 @@ type parserStorage      is [@layout:comb] record[
   assetName               : big_map(tokenId, assetString);
   assetId                 : big_map(assetString, tokenId);
   metadata                : big_map(string, bytes);
+  tmp                     : option(tmpPrice);
```

Comment:
Maybe using option(bytes) is better. Than we will be able to connect more oracles with different needs